include/seqan3/range/container/small_vector.hpp

```diff
@@ -920,7 +920,7 @@ class small_vector
     }
     //!\}
 
-protected:
+public:
     //!\privatesection
```

Comment:
For the change to public interface I suggest a few further actions:
1. Is `\privatesection` still valid? To me it looks like it should be removed. 
2. Rename `sz` and `data_` to `size` and `data`, as they are public now.
3. Remove the `public:` specifier below, it is duplicated.

Reply:
we already have public members `data` and `size`.

`\privatesection` is still valid.

Reply:
I don't understand why it is still valid, but if you know what you're doing I'm fine ;)

Reply:
`sz` and `data_` are still private, because our recommended way to access that data is `size()` and `data()`. So we should not advertise the "private" members in our documentation.

Reply:
Okay, you can fix only point 3 and then I approve :)

Reply:
The `public:` below re-enables public documentation. If we merge them I would need to add a `//!\publicsection` comment. I would prefer to keep `public:` instead.

Reply:
okay, then we can keep it...

Reply:
I agree with point 3. of joerg but I don't care much